percy-csharp-selenium/Percy.cs

```diff
@@ -161,6 +175,38 @@ public void Snapshot(String name, List<int> widths, int minHeight, bool enableJa
             PostSnapshot(domSnapshot, name, widths, minHeight, driver.Url, enableJavaScript, percyCSS);
         }
 
+        /**
+        * Attempts to load dom.js from the local Percy server. Use cached value in `domJs`,
+        * if it exists.
+        *
+        * This JavaScript is critical for capturing snapshots. It serializes and captures
+        * the DOM. Without it, snapshots cannot be captured.
+        */
+        private async Task<string> FetchPercyDOM() {
+        
+            try {
+                HttpResponseMessage response = await client.GetAsync(PERCY_SERVER_ADDRESS + "/percy/dom.js");
+                int statusCode = (int)response.StatusCode;
+
+                if (statusCode != 200){
+                    throw new Exception("Failed with HTTP error code: " + statusCode);
+                }
+
+
+                HttpContent httpEntity = response.Content;
+                String domString = httpEntity.ReadAsStringAsync().Result;
+                domJs = domString;
+
+                return domString;
+            } catch (Exception ex) {
+                isPercyEnabled = false;
+                Console.WriteLine("[percy] Something went wrong attempting to fetch DOM: " + ex.Message);
+
```

Comment:
we need to have this in the MVP. Debugging info is crucial for us when this goes customer facing.

```
 if (PERCY_DEBUG) { log(e.getMessage()); }
```